snow/consensus/snowball/tree_test.go

```diff
@@ -632,8 +576,8 @@ func TestSnowballResetSibling(t *testing.T) {
 		expected := `SB(Preference = 0, PreferenceStrength[0] = 2, PreferenceStrength[1] = 1, SF(Confidence = 1, Finalized = false, SL(Preference = 0))) Bit = 0
     SB(Preference = 1, PreferenceStrength[0] = 0, PreferenceStrength[1] = 2, SF(Confidence = 1, Finalized = false, SL(Preference = 1))) Bit = 1
         SB(PreferenceStrength = 0, SF(Confidence = 0, Finalized = false)) Bits = [2, 256)
-        SB(PreferenceStrength = 2, SF(Confidence = 1, Finalized = true)) Bits = [2, 256)
-    SB(PreferenceStrength = 1, SF(Confidence = 1, Finalized = true)) Bits = [1, 256)`
+        SB(PreferenceStrength = 2, SF(Confidence = 1, Finalized = false)) Bits = [2, 256)
```

Comment:
same comment as above. In this case the snowball preference strength is 2, but the confidence counter towards commit is still 1.